pythorhead/mention.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, Optional
+from pythorhead.requestor import Request, Requestor
+from pythorhead.types import SortType
+
+
+class Mention:
+    def __init__(self, _requestor: Requestor):
+        self._requestor = _requestor
+
+    def list(
+        self,
+        unread_only: bool,
+        sort: Optional[SortType] = None,
+        page: Optional[int] = None,
+        limit: Optional[int] = None,
+    ) -> Optional[dict]:
+        """
+        List all user mentions
+
+        Args: 
+            unread_only (bool).
+            sort? (SortType)
+            page? (int).
+            limit? (int)
+
+        Returns:
+            dict? mentions response
+        """
+
+        params = {"auth": self._requestor.get_auth_token()}
```

Comment:
auth is automatically set by requestor, you don't need to do that here

Reply:
Okay, I figured out the issue. I should not have used `json` and `params` for a GET request, `params` is enough. I've fixed this and removed the extra methods I created to help fetch the token.